ddtrace/tracer.py

```diff
@@ -191,6 +192,7 @@ async def web_handler(request):
         return self.context_provider.active(*args, **kwargs)
 
     # TODO: deprecate this method and make sure users create a new tracer if they need different parameters
+    @debtcollector.removals.removed_kwarg("collect_metrics", removal_version="0.49")
```

Comment:
Might be worth having a `message` here for the replacement behaviour instead of the release note.